en.json

```diff
@@ -1,12 +1,12 @@
 {
   "Learn More" : "Learn More",
   "CLOSED CAPTION PREVIEW": "CLOSED CAPTION PREVIEW",
-  "Sample Text": "Sample Text",
+  "Sample Text": "Hi, I'm a caption!",
   "Ad": "Ad",
   "Skip Ad": "Skip Ad",
   "LIVE": "LIVE",
   "GO LIVE": "GO LIVE",
-  "CC Options": "CC Options",
+  "CC Options": "Closed captions",
```

Comment:
Same as above, "CC Options" is already translated so I figure we just leave it as that for now until we get product to say that want it to say something else in that language.